overlay_ports/commonlibsse-ng-vr/patches/05-tesdatahandler-add_vr_support.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/include/RE/T/TESDataHandler.h b/include/RE/T/TESDataHandler.h
+index 21293e60..4a314aed 100644
+--- a/include/RE/T/TESDataHandler.h
++++ b/include/RE/T/TESDataHandler.h
+@@ -38,6 +38,9 @@ namespace RE
+ 	class TESDataHandler : public BSTSingletonSDM<TESDataHandler>
+ 	{
+ 	public:
++#ifdef ENABLE_SKYRIM_VR
++		inline static RE::TESFileCollection* VRcompiledFileCollection = nullptr;  // used by SkyrimVRESL to store pointer to VR version
```

Comment:
I might not really understand what this is about but why inline?

Reply:
It is needed because `TESDataHandler->compiledFileCollection` isn't available on SkyrimVR, but SkyrimVRESL provides an alternative as `VRcompiledFileCollection`. I haven't tested it but at least patching it made it compile and run.

As for why it has to be inline... I honestly have no idea 😅 I'm not really a C++ developer, and I guess I copied it from somewhere.